pkg/apimachinery/utils/meta.go

```diff
@@ -751,14 +632,23 @@ func (m *grafanaMetaAccessor) GetManagerProperties() (ManagerProperties, bool) {
 	res := ManagerProperties{
 		Identity:    "",
 		Kind:        ManagerKindUnknown,
-		AllowsEdits: true,
+		AllowsEdits: false,
```

Comment:
@radiohead -- IIUC, anything managed should be read only *unless* we explicitly allow edits.  Either via the allowEdits annotation, or by looking up more data based on the id (eg in the repo case)

For something like "plugin" (the values statically embedded in the distribution) allowEdits can never be true

Reply:
SGTM